config/test/config.go

```diff
@@ -19,7 +20,23 @@ func DefaultConfigForTesting() *config.Config {
 	return cfg
 }
 
-// DefaultProfileForTesting constructs a profile with precomputed keys, only used for testing.
+// DefaultMemConfigForTesting constructs a config with precomputed keys & settings for an in memory repo & fs, only used for testing.
+func DefaultMemConfigForTesting() *config.Config {
```

Comment:
It's possible we want this to be the `DefaultConfigForTesting`, but separated it out in case we wanted to split the concerns.

Reply:
We now have a `test` package for Config details. Let's move this into `github.com/qri-io/qri/config/test`

See 1aeba849ee7b88c4e44cfbf7c1d4eb065c505cc3 for the commit that makes the change, and the rational behind a separate test configuration package

I'm a fan of the `Mem` differentiator. We do have tests that need to be stuff other than mem

Reply:
this is in the config test package! :)